scanapi/utils.py

```diff
@@ -35,3 +41,17 @@ def _validate_required_keys(keys, required_keys, scope):
     if not set(required_keys) <= set(keys):
         missing_keys = set(required_keys) - set(keys)
         raise MissingMandatoryKeyError(missing_keys, scope)
+
+
+def session_with_retry(retry_configuration):
```

Comment:
@flaviotruzzi can we have a docstring to indicate what is this method for? The docstring is used to build our internal code documentation. I will add to the wiki that we need a docstring for every new method, this way new comers will know that too.